resources/views/accounts/_edit_notifications.blade.php

```diff
@@ -26,20 +26,15 @@
         <div class="account-edit__input-group">
             <div class="account-edit-entry account-edit-entry--no-label js-account-edit" data-account-edit-auto-submit="1" data-skip-ajax-error-popup="1">
                 <label class="account-edit-entry__checkbox">
-                    <div class="osu-checkbox">
-                        <input
-                            name="user[user_notify]"
-                            class="osu-checkbox__input js-account-edit__input"
-                            type="checkbox"
-                            @if (Auth::user()->user_notify)
-                                checked
-                            @endif
-                        >
-                        <span class="osu-checkbox__box"></span>
-                        <span class="osu-checkbox__tick">
-                            <i class="fas fa-check"></i>
-                        </span>
-                    </div>
+                    @include('objects._switch', [
+                        'attributes' => [
+                            'name' => 'user[user_notify]',
+                            'type' => 'checkbox',
```

Comment:
what do you think about just defaulting to `checkbox` with an option to turn it into radio? saves having to pass through `'type' = 'checkbox'` in all of the places